components/Navigation/Navigation.themes.ts

```diff
@@ -5,37 +5,40 @@ export namespace Theme {
     navBg: Property.Color;
     navButtonBg: Property.Color;
     navButtonHoverBg: Property.Color;
-    navButtonActiveBg: Property.Color;
+    navButtonHoverBg2: Property.Color;
     navButtonFocusBg: Property.Color;
+    navButtonFocusBg2: Property.Color;
     navButtonFocusBorder: Property.Color;
     navButtonText: Property.Color;
     navButtonHoverText: Property.Color;
     navButtonActiveText: Property.Color;
     navButtonFocusText: Property.Color;
   };
 
-  type Factory = (primaryColor: Property.Color) => Colors;
+  type Factory = (primaryColor?: Property.Color) => Colors;
 
-  export const getLight: Factory = (primaryColor) => ({
+  export const getLight: Factory = () => ({
     navBg: 'white',
     navButtonBg: 'transparent',
-    navButtonHoverBg: 'hsla(0, 0%, 0%, 0.04)',
-    navButtonActiveBg: 'hsla(0, 0%, 0%, 0.04)',
-    navButtonFocusBg: tinycolor(primaryColor).saturate(27).lighten(58).setAlpha(0.33).toString(),
-    navButtonFocusBorder: tinycolor(primaryColor).lighten(12).toString(),
-    navButtonText: 'hsla(0, 0%, 0%, 0.54)',
+    navButtonHoverBg: tinycolor('black').setAlpha(0.04).toHslString(),
+    navButtonHoverBg2: 'transparent',
+    navButtonFocusBg: tinycolor('white').setAlpha(0.15).toHslString(),
+    navButtonFocusBg2: '$primary',
+    navButtonFocusBorder: '$neon8',
```

Comment:
@matthieuh as the theme allows using other primary colors in the provider, the users can, for example, set the primary color to `blue`. Do you think it would make sense to use `neon` here anyways?

Reply:
I pushed a commit to make it way easier to get primary color variations. I would like your input on it :)